Project-Aurora/Project-Aurora/Devices/SteelSeries/GameSenseSDK.cs

```diff
@@ -0,0 +1,215 @@
+        }
+
+        public void setHeadsetColor(byte red, byte green, byte blue)
+        {
+            sendColor("HEADSETCOLOR", red, green, blue);
+        }
+
+        public void sendColor(String deviceEvent, byte red, byte green, byte blue)
+        {
+            GameSensePayloadPeripheryColorEventJSON payload = new GameSensePayloadPeripheryColorEventJSON();
+            payload.game = sseGameName;
+            payload.Event = deviceEvent;
+            payload.data = "{\"value\": [" + red + ", " + green + ", " + blue + "]}";
+            // sending POST request
+            String json = JsonConvert.SerializeObject(payload);
+            sendPostRequest("http://" + sseAddress + "/game_event", json);
+        }
+
+        public void setKeyboardColors(List<byte> hids, List<Tuple<byte, byte, byte>> colors)
+        {
+            GameSensePayloadPeripheryColorEventJSON payload = new GameSensePayloadPeripheryColorEventJSON();
+            payload.game = sseGameName;
+            payload.Event = "KEYBOARDCOLORS";
+
+            payload.data = "{";
+            payload.data += "\"hids\": ";
+            payload.data += JsonConvert.SerializeObject(hids);
+            payload.data += ",";
+            payload.data += "\"colors\": [";
+            foreach (Tuple<byte, byte, byte> color in colors)
+            {
+                payload.data += "[" + color.Item1 + ", " + color.Item2 + ", " + color.Item3 + "], ";
+            }
+            payload.data += "]";
+            payload.data += "}";
+
+            // sending POST request
+            String json = JsonConvert.SerializeObject(payload);
+            sendPostRequest("http://" + sseAddress + "/game_event", json);
+        }
+
+        public void sendHeartbeat()
+        {
+            GameSensePayloadHeartbeatJSON payload = new GameSensePayloadHeartbeatJSON();
+            payload.game = sseGameName;
+            // sending POST request
+            String json = JsonConvert.SerializeObject(payload);
+            sendPostRequest("http://" + sseAddress + "/game_heartbeat", json);
+        }
+
+        public void sendStop()
+        {
+            GameSensePayloadPeripheryColorEventJSON payload = new GameSensePayloadPeripheryColorEventJSON();
+            payload.game = sseGameName;
+            payload.Event = "STOP";
+            // sending POST request
+            String json = JsonConvert.SerializeObject(payload);
+            sendPostRequest("http://" + sseAddress + "/game_event", json);
+        }
+
+        private void setupLISPHandlers()
+        {
+            String json = "";
+            GameSensePayloadLISPHandlerJSON payload = new GameSensePayloadLISPHandlerJSON();
+            payload.game = sseGameName;
+
+            // sending POST requests with golisp handler
+            payload.golisp = "(handler \"PERIPHERYCOLOR\"  (lambda (data)    (let* ((val (value: data)))      (on-device \"rgb-2-zone\" show: val) (on-device \"rgb-1-zone\" show: val))))";
```

Comment:
The GameSense API expects all the GoLisp handlers to be in one file, which this call creates. Because you call them every time they launch, the left-over handlers are still used from the previous calls so it still works but putting all these handles into one string and one call to `load_golisp_handlers` is the correct way to do it.